### PR TITLE
[04/08/20] Added information about ROS Parameters

### PR DESCRIPTION
**Pull request by adbidwai**
I have currently added this as a separate markdown file to avoid conflict issues. Please place it wherever necessary.
Also feel free to suggest any changes that need to be done in this document.

### CODE REVIEW

File: docs/automation/ROS/ros_p2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Parameter server is a collection of values or parameters that can be retrieved or modified by the nodes during runtime upon requests through command prompt, nodes or launch files. Parameters are intended to be static, globally available values like integers, floats, strings and boolen values and can be stored independently or within a YAML file. Parameters are meant to be gloablly viewable so nodes can easily inspect the configuration state of the system and modify if necessary.
+
+### Accessing and setting Parameters
+#### Via command line 
+Parameters can be accessed, modified or deleted using the `rosparam` command line utility in the `rosbash` suite of terminal commands.
+1) To list all the parameters : <br/> 
+    ```rosparam list```
+    
+   Or to list all the parameters in a specific namespace : <br/>
```
Comment (veds12) [inline docs/automation/ROS/ros_p2.md]: Can you add the ros wiki namespace link too?

File: docs/automation/ROS/ros_p2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2) To assign a value to an already existing parameter or to set a new one : <br/>
+    ```rosparam set <parameter_name> <parameter_value>```
+    
+    Note : You can also load the parameters into the parameter server from a YAML file using <br/>
+    ```rosparam load <filename> <namepsace>```
+    
+3) To get/read a parameter value : <br/>
+    ```rosparam get <parameter_name>```
+    
+   Note : You can also dump/save the parameters into a YAML file from parameter server using <br/>
```
Comment (veds12) [inline docs/automation/ROS/ros_p2.md:23]: Can you also briefly explain parameter server? Or put a link for it?

**Review by veds12 — CHANGES_REQUESTED** [review]

**Comment by veds12** [comment]
Change the PR message to the format `[04/08/20] Commit Message` so that the Changelog is readable. Also, you will need to add this file's name in the `nav` section of the mkdocs.yml file for the website to deploy it. 

```
nav:
   ...
   - Automation:
        ...
        - ROS:
            ...
            - ROS Parameters and Parameter Server: automation/ROS/ros_p2.md
            ...
```

File: docs/automation/ROS/ros_p2.md (hunk 2 above)
Comment (adbidwai) [reply docs/automation/ROS/ros_p2.md:23]: Parameter server is briefly explained in the first paragraph

File: docs/automation/ROS/ros_p2.md (hunk 1 above)
Comment (adbidwai) [reply docs/automation/ROS/ros_p2.md]: Cool, I'll add a resource

File: docs/automation/ROS/ros_p2.md (hunk 2 above)
Comment (veds12) [reply docs/automation/ROS/ros_p2.md:23]: Oh ok didn't see that

**Comment by adbidwai** [comment]
All request changes done.

**Pull request merged**